packages/dx-grid-core/src/plugins/custom-tree-data/computeds.ts

```diff
@@ -70,7 +70,7 @@ export const expandedTreeRows: ExpandedTreeRowsFn = (
 
   let currentExpanded = true;
   let currentLevel = 0;
-  return rows.reduce((acc, row) => {
+  return Array.prototype.slice.call(rows).reduce((acc, row) => {
```

Comment:
Is it required?

Reply:
Yeah, `seamless-immutable` lib. replace all basic array methods. I have created tests to show it needs.